README.md

```diff
@@ -50,14 +46,13 @@ Required options:
   - `--s3-bucket`: The S3 bucket containing your org's kubectl config file
   - `--s3-key`: The S3 key of your org's kubectl config file
 
-### Initializing a project
+### Setting up an existing project
 
-* `hokusai init` - Writes hokusai project config to `hokusai/config.yml`, creates test, development and production yaml files alongside it, and adds a Dockerfile to the current directory.
+* `hokusai setup` - Writes hokusai project config to `hokusai/config.yml`, creates test, development and production yml files alongside it, adds a Dockerfile to the current directory, and creates a project ECR repo.
```

Comment:
isn't it actually called 'yaml'? seems more human readable than the file extension.